src/main/java/edu/ucsb/cs156/gauchoride/entities/Shift.java

```diff
@@ -0,0 +1,26 @@
+package edu.ucsb.cs156.gauchoride.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.AccessLevel;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder
+@Entity(name = "shift")
+public class Shift {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private long id;
+  private String day;
+  private String shift;
+  private String driver;
+  private String driverBackup;
```

Comment:
I wonder if these should be userid's instead of Strings?

With the userid, we can refer to the driver's name, email, and any other information we might associate with the driver in the future in the application.

With a String, all we have is a string.

Reply:
I think that's a good idea. I can replace those 2 Strings with the userIds for the main and backup driver

Reply:
I'll also add the repository file as well.